tests/contrib/sensors/test_redis_pub_sub_sensor.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import unittest
+from airflow import DAG, configuration
+from airflow.contrib.sensors.redis_pub_sub_sensor import RedisPubSubSensor
+from airflow.utils import timezone
+from mock import patch, call, MagicMock
+
+DEFAULT_DATE = timezone.datetime(2017, 1, 1)
+
+
+class TestRedisPubSubSensor(unittest.TestCase):
+
+    @patch('airflow.contrib.hooks.redis_hook.RedisHook.get_conn')
```

Comment:
In the CI actually runs a Redis using docker-compose. Maybe you can do end to end testing instead of mocking. This makes it easier when you want to test against newer versions of Redis. What do you think?

Reply:
@Fokko - 

I see that the redis key sensor uses an integration test and will try your recommendation.  

Thanks.

Reply:
@Fokko - I've added integration tests based on your recommendations.  Thanks for your advice.